app/requests/WoW/UploadTimers.ts

```diff
@@ -0,0 +1,28 @@
+import { address, UserAgent } from '~/requests/client/config'
+
+export interface UploadTimersResponse {
+  data: Array<UploadTimersItem>
+}
+
+export interface UploadTimersItem {
+  dataSetId: number
+  dataSetName: Array<string>
+  lastUploadMinute: number
+  lastUploadTimeRaw: string
+  lastUploadUnix: string
+  region: string
+  tableName: string
+}
+
+const UploadTimers: () => Promise<Response> = async () => {
+  return fetch(`${address}/api/wow/upload-timers`, {
```

Comment:
I think your problem might be that the api address is this:
`/api/wow/uploadtimers`

Reply:
That did the trick! Sorry for the inconvenience 😓

Reply:
I did something like this in my current wip branch that might work the same here:

`  <p className=" px-3 py-2 max-w-[200px] overflow-x-scroll">
        {row.dataSetName.join(', ')}
      </p>`

Reply:
![image](https://github.com/ff14-advanced-market-search/saddlebag-with-pockets/assets/18511031/a5366bc6-d2f5-474b-9dd4-7739ab87bbef)

Here's what I did - not sure if you meant commas or newlines, but I think it looks pretty nice. (and similar to the temp site!)